projections/gorm.go

```diff
@@ -427,6 +427,21 @@ func (p *GORMDB) GetContentEntities(ctx context.Context, entityFactory weos.Enti
 	return entities, count, result.Error
 }
 
+func (p *GORMDB) GetByIdentifiers(ctxt context.Context, entityFactory weos.EntityFactory, identifiers map[string]interface{}) ([]map[string]interface{}, error) {
```

Comment:
Why isn't GetByKey used?

Reply:
Get key is specific to primary keys.  I believe this was brought up in planning where the getbykey function ensures the values are primary key values.  I can change the function names to make this clearer.